tests/unit/test_openstack.py

```diff
@@ -603,7 +603,8 @@ async def test_network_picking_no_spread(self, x):
         )
 
     @pytest.mark.asyncio
-    async def test_openstack_gather_responses(self):
+    @patch("asyncio.sleep", new_callable=AsyncMock)
```

Comment:
I guess the patch of `asyncio.sleep` is done because of the usage in `._openstack_gather_responses. But it also makes its calls in `coro1`, `coro2`, `coro3` not doing anything so the `asyncio.sleep` can be removed from those coros.